designer/submodules/packages/components/src/components/usages/componentView.tsx

```diff
@@ -54,6 +55,14 @@ function Component(): JSX.Element {
                             return t("table.filter.SCENARIOS", "Scenarios");
                     }
                     break;
+                case "USAGE_TYPE":
+                    switch (value) {
+                        case UsagesFiltersUsageType.INDIRECT:
+                            return t("table.filter.INDIRECT", "Indirect usage");
+                        case UsagesFiltersUsageType.STRAIGHT:
+                            return t("table.filter.STRAIGHT", "Straight usage");
```

Comment:
Straight is the correct word for that? I think "direct" sounds better. Let's keep this simple.

Reply:
Agree - direct/indirect would fit better here.